perma_web/api/authorizations.py

```diff
@@ -17,7 +17,9 @@ def can_access(self, user, obj):
         # shared folders
         elif obj.vesting_org_id:
             if user.is_vesting_org_member():
-                return user.vesting_org_id == obj.vesting_org_id
+                for vo in user.vesting_org.all():
```

Comment:
Ideally we would rename `vesting_org` to `vesting_orgs`. Up for a big, careful find-and-replace?